active_directory_ldap/komand_active_directory_ldap/connection/connection.py

```diff
@@ -85,14 +88,23 @@ def host_formatter(self, host: str) -> str:
         """
         colons = host.count(':')
         if colons > 0:
-            self.logger.info('Port was provided in hostname, using value from Port field instead')
             host = host.split(':')
             if colons == 1:
-                host = host[0]
+                if host[1].find('//') != -1:
```

Comment:
This looks good but I wonder if there's a way to implement this as input validation in the front-end instead of silently modifying the user input and discarding data?  For example, if the user puts in 636 on the port but provides the ldap url of ldap://whatever.contoso.com:389 in the server, it'll try and connect to whatever.contoso.com:636 but the interface won't reflect this behavior.  It'll only be visible in the logs.
If implementing it in a more transparent way isn't possible, this is an absolutely great addition to help handle weird input issues.

Reply:
We will be adding more input validation in the FE this year.